### ISSUE TITLE
"Shorten URL" button has insufficient contrast ratio for accessibility purposes

### ISSUE THREAD

**Issue by sarahesbie** [origranot/reduced.to#165]
The color combination #ffffff/#3498db has a contrast ratio of 3.15, which is not sufficient. At this size, you will need a ratio of at least 4.5.

Consider using the following foreground/background combination:

#ffffff/#007dbb  (has a ratio of 4.52)

![URL shortener contrast ratio](https://user-images.githubusercontent.com/4270573/195077773-ce6e2e8a-7119-458a-becb-1e38dd61dfc0.jpg)

More info on contrast and accessibility: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast

**Comment by cweave** [comment]
One of the changes I made for the button would help: https://github.com/origranot/url-shortener/issues/172#issuecomment-1275532432

I pulled one of the blue colors `#0069d9` to use as the background, with white text, that brings it to 5.22:1

![image](https://user-images.githubusercontent.com/16121690/195243436-d30ed672-7c7e-4dc3-bd4a-54cd2c6c2f8a.png)

The lighter blue, `#3498db` I used on hover, however, would be a no go.

![image](https://user-images.githubusercontent.com/16121690/195243535-8e2909be-2f04-4dea-be32-78a1175be9cb.png)

Just some examples :)
